src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -52,567 +51,718 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
-	                //   1. The account has been blanked out by the user
-	                //   2. The Account Model is not blank
-	                if (c.AccountId == null && defaultRecTypeID != null) {
-	                    listContactNeedAccount.add(c);
-	                }
-
-                    // if the account (any type) has changed
-                    if (c.AccountId != oldContacts[i].AccountId) {
-                        if (c.AccountId != null) {
-                            listAccountIdHHToUpdate.add(c.AccountId);
-
-                            //Used to Adv to inject oppor logic
-                            mapContactIdAccountIdNewMoveOpps.put(c.Id, c.AccountId);
-
-                            if (checkHouseholdNaming(c.Account.RecordTypeId)) {
-                                setAccountIdHHToRename.add(c.AccountId);
-                                if (oldContacts[i].AccountId != null) {
-                                    setAccountIdHHToRename.add(oldContacts[i].AccountId);
-                                }
-                            }
-                        }else{
-                            if (UTIL_CustomSettingsFacade.getSettings().Automatic_Household_Naming__c == true) {
-                                setAccountIdHHToRename.add(oldContacts[i].AccountId);
-                            }
-                        }
+        if (triggerAction == TDTM_Runnable.Action.AfterInsert) {
+            Map<Id,Id> updateAccPrimaryContact = insertWrapper.accIdConIdToUpdate;
+            Set<Id> accountIdHHToRename = insertWrapper.accountIdToRename;
+
+            //Updates the Primary Contact Field on New Account
+            if (updateAccPrimaryContact.size() > 0) {
+                updateAccountPrimaryContact(updateAccPrimaryContact, dmlWrapper);
+            }
 
-                        // old HH Accounts that don't get deleted still need to have name redone
-                        if (oldContacts[i].AccountId != null) {
-                            listAccountIdHHToUpdate.add(oldContacts[i].AccountId);
+            //Updates Household Account name
+            if (accountIdHHToRename.size() > 0) {
+                UTIL_ACCT_Naming.updateHHAccountsName(accountIdHHToRename);
+            }
 
-                            //Used to Adv to inject oppor logic
-                            mapContactIdAccountIdOldMoveOpps.put(c.Id, oldContacts[i].AccountId);
-                        }
-                    }
+            TDTM_TriggerHandler.processDML(dmlWrapper, true);
+            updateRecursionFlag();
+        }
+    }
 
-                    // check for ownership changes
-                    if (c.OwnerId != oldContacts[i].OwnerId) {
-                    	mapContactIdContactOwnerChange.put(c.Id, c);
-                    }
+    /*******************************************************************************************************
+    * @description Processes after update logic.
+    * @param c is a Contact record from trigger new.
```

Comment:
Minor, I think all of these ApexDocs were meant for the HandleUpdateWrapperLogic() constructor below.